util/ci/check_png_optimized.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -e
+
+optimized=0
+
+for file in $(git ls-files *.png); do
```

Comment:
```suggestion
git ls-files "*.png" | sort -u | while read file; do
```

Reply:
This does not work, since pipes create a subshell where `optimized` is not available anymore, but I applied your suggestion and made it `optimized=$(...)` instead.

I hope this is appropriate.